capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capatalize
```

Comment:
- [ ] There's a typo in this `capitalize` method. Please consider fixing it! 🤓 🔧 

```suggestion
    @nameable.correct_name.capitalize
```